agent/configentry/resolve_test.go

```diff
@@ -24,6 +24,26 @@ func Test_ComputeResolvedServiceConfig(t *testing.T) {
 		args args
 		want *structs.ServiceConfigResponse
 	}{
+		{
+			name: "proxy with balanceinboundconnections",
+			args: args{
+				scReq: &structs.ServiceConfigRequest{
+					Name: "sid",
+				},
+				entries: &ResolvedServiceConfigSet{
+					ServiceDefaults: map[structs.ServiceID]*structs.ServiceConfigEntry{
+						sid: {
+							BalanceInboundConnections: "exact_balance",
+						},
+					},
+				},
+			},
+			want: &structs.ServiceConfigResponse{
+				ProxyConfig: map[string]interface{}{
+					"balance_inbound_connections": "exact_balance",
+				},
+			},
+		},
```

Comment:
Are you referring to the test or the field itself? I believe that test will throw an error if it doesn't see "exact_balance".

Since this configuration is going onto the global service-defaults objects, I decided to not have an error thrown if it sees an unknown value to prevent future options from causing problems. I suppose I could have it log a warning message, though.

Reply:
Sorry, it's still too early 😂  I meant to link to agent/structs/config_entry_test.go. Given that there is only one valid value, it would be nice to give feedback when users save configuration entries.